package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@tef-novum/webview-bridge",
-    "version": "2.1.0",
+    "version": "2.2.0-beta.1",
```

Comment:
hmm, why is it on beta? 😮

Reply:
this way I can test it without making a release, package is published with `--tag beta`